synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -259,6 +277,28 @@ void CanvasResize::on_height_changed()
 	refresh_wh_toggle_widgets();
 }
 
+void CanvasResize::on_size_template_changed(){
+	String selection(combo_box->get_active_text());
+	if(selection == "Custom Size"){
```

Comment:
It won't work with the translated string. You should set the combobox item id (with a non translatable string) and check it here.